docs/docs/nlu/choosing-a-pipeline.mdx

```diff
@@ -30,21 +30,15 @@ components you want to use directly in the configuration file. See
 
 ### The Short Answer
 
-If your training data is in English, a good starting point is the following pipeline:
-
-```yaml (docs/sources/data/configs_for_docs/default_english_config.yml)
-```
-
-If your training data is not in English, start with the following pipeline:
-
-```yaml (docs/sources/data/configs_for_docs/default_config.yml)
-```
+To get started, you can let the
+[Suggested Config](../api/model-configuration.mdx#suggested-config) feature choose a
+default pipeline for you.
```

Comment:
Do you have to fill in the language parameter to do so? If so i'd specify that, and show an example: 
```
language: fr  # your 2-letter language code

pipeline:
# intentionally left empty
```

(i don't really know the code of how it chooses a pipeline for you or if it just chooses the language-agnostic one)

Reply:
I like the idea of providing examples. Will add one to the main documentation for this feature, too.